03.bowling/bowling.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+score = ARGV[0]
+scores = score.split(',')
+
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames_tmp = shots.each_slice(2).to_a
+pins_of_tenth = frames_tmp.slice(9..-1)
+frame_of_tenth = []
```

Comment:
10回目 = 最終、と見なして、

```suggestion
last_pins = frames_tmp.slice(9..-1)
last_frame = []
```

とするのはどうでしょう？
（`of`ってあまりプログラミングで使わないんですよね。ゼロではないですが）

Reply:
その変数名も悩んだところでした……
last, first を積極的に使っていくようにします！